script/publish

```diff
@@ -78,6 +82,11 @@ function uploadOSXAssets() {
   return Promise.all(uploads)
 }
 
+function uploadLinuxAssets() {
+  const uploads = [upload(distInfo.getLinuxZipName(), distInfo.getLinuxZipPath())]
```

Comment:
This first file here is invalid, because it's just the file path. Again, which assets are we trying to upload here?